### PR TITLE
ci: introduce skywalking-eyes

### PR DESCRIPTION
**Pull request by dmsolr**
Please answer these questions before submitting a pull request

Introduce apache/skywalking-eyes to do license header check. 

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### CODE REVIEW

File: conf/kubeconfig [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 apiVersion: v1
 clusters:
 - cluster:
```
Comment (dmsolr) [inline conf/kubeconfig:18]: Please double check this file whether we can need a license header?

File: .licenserc.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - '**/assets/lcs-templates/**'
+    - '**/assets/languages.yaml'
+    - '**/assets/assets.gen.go'
+    - '**/*.json'
+    - '.github/ISSUE_TEMPLATE'
+    - '.github/PULL_REQUEST_TEMPLATE'
+    - 'pkg/kube/apisix/client/**'
+    - '**/zz_generated.deepcopy.go'
+
+  comment: on-failure
```
Comment (tokers) [inline .licenserc.yaml:41]: Please check the editor config, the EOL symbol is required.

File: conf/kubeconfig (hunk 1 above)
Comment (dmsolr) [reply conf/kubeconfig:18]: I am asking the file whether we can add a license header. 
If it is not in use, I think it should be removed in a new PR.

**Review by tokers — APPROVED** [review]

File: .github/workflows/license-checker.yml [hunk 3]
```diff
@@ -14,7 +32,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
-      - name: run license check
-        run: |
-          make license-check
+      - name: Check License Header
+        uses: apache/skywalking-eyes@v0.1.0
```
Comment (heyanlong) [inline .github/workflows/license-checker.yml:36]: Maybe it is more appropriate to use `apache/skywalking-eyes@main` here

**Comment by heyanlong** [comment]
Please check if read and write permissions are set

`Settings` -> `Actions` -> `Runners` -> `Workflow permissions`

**Comment by dmsolr** [comment]
> Please check if read and write permissions are set
> 
> `Settings` -> `Actions` -> `Runners` -> `Workflow permissions`

Thank you, @heyanlong .
The author of skywalking-eyes told me, because of github action limitation, auto-comment bot cannot work while the PR from the fork repo.

File: .github/workflows/license-checker.yml (hunk 3 above)
Comment (dmsolr) [reply .github/workflows/license-checker.yml:36]: fixed by #510

**Pull request merged**